pyproject.toml

```diff
@@ -35,7 +35,7 @@ poetry = "^1.7.1"
 poetry-plugin-export = "^1.6.0"
 functions-framework = "^3.5.0"
 cron-validator = "^1.0.8"
-prediction-market-agent-tooling = "^0.13.1"
+prediction-market-agent-tooling = { version = "^0.13.1", extras = ["langchain", "google"] }
```

Comment:
You can use `extras = ["all"]` to save specifying them manually
https://python-poetry.org/docs/pyproject/#extras

Reply:
it seems like it should work, but if I do it, it removes them from the lock file:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/d9524140-e398-4a20-b6a8-0d73e85cb9bf)

and after removing and installing the poetry env again, it's not there:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/1af0b507-e4a1-486b-a761-53c3d1aa3907)

but if I keep it explicit it works:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/c2c577fa-c6a4-4439-9651-78232ab15969)